core/server/src/main/java/alluxio/master/file/meta/options/CreatePathOptions.java

```diff
@@ -90,6 +92,16 @@ public Builder setDirectory(boolean directory) {
     }
 
     /**
+     * @param mountPoint the mount point flag to use; it specifies whether the object to create is
```

Comment:
not sure I understand your comment ... the mount function ends up invoking loadMetadataDirectory, which ends up invoking mkdir, which ends up invoking createPath which uses these options ... we need to propagate the "mount point" bit from loadMetadataDirectory to createPath.